packages/models/organizations.coffee

```diff
@@ -7,8 +7,13 @@ Organization = Astro.Class
     name: 'string'
     description: 'string'
     createdById: 'string'
-
   methods:
+    addMember: (userId) ->
+      UserProfiles.update({userId: userId}, {
+        $addToSet: {memberOfOrgs: @_id}
+      })
```

Comment:
I think that's a bit like declaring methods as public or private in Java or C++. It makes the programmer's intent more explicit at the cost of making the code more verbose and less flexible. I don't have a strong preference in either direction, so I went ahead and made the method server-only since that seems to be your preference.